.github/workflows/codecov.yaml

```
@@ -19,20 +19,23 @@ jobs:
 
       - run: just submodules
       - run: zig build
       - run: just coverage
-      - uses: codecov/codecov-action@v7
+      - uses: codecov/codecov-action@v4
         with:
           verbose: true
           token: ${{ secrets.CODECOV_TOKEN }}
+          disable_search: true
+          files: ./.coverage/all/kcov-merged/cobertura.xml
+          fail_on_error: true
```

actionlint (1.7.12)

[error] input "fail_on_error" is not defined in action "codecov/codecov-action@v4". available inputs are "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "file", "files", "flags", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugin", "plugins", "report_code", "root_dir", "slug", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)